.circleci/config.yml

```diff
@@ -78,4 +78,33 @@ workflows:
             - "Publish development orb"
           filters:
             branches:
-              only: master
\ No newline at end of file
+              only: master
+
+  weekly-dev-publish:
+    triggers:
+      - schedule:
+          cron: "0 8 * * 1"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - orb-tools/publish:  &publish
+          pre-steps:
+            - checkout
+            - run:
+                name: Pack the orb
+                command: circleci config pack orb/ > orb.yml
+
+          name: "Publish development orb"
+          orb-path: orb.yml
+          orb-ref: silta/silta@dev:${CIRCLE_BRANCH}
+          publish-token-variable: "$CIRCLECI_DEV_API_TOKEN"
+          validate: true
+
+      - validate:
+          requires:
+            - "Publish development orb"
+          filters:
+            branches:
+              only: master
```

Comment:
This work well, but to reduce duplicate code let's change like this:

```
      - validate: &validate
          requires:
            - "Publish development orb"
          filters:
            branches:
              only: master

  weekly-dev-publish:
    triggers:
      - schedule:
          cron: "0 8 * * 1"
          filters:
            branches:
              only:
                - master
    jobs:
      - orb-tools/publish: *publish

      - validate: *validate
```

Reply:
This is an excellent idea! Done!!